default.json5

```diff
@@ -150,12 +146,6 @@
         "patch"
       ]
     },
-    // has to be placed after the general grouping rules above
```

Comment:
In hindsight, there was no reason to create a separate group for these updates. We just needed to make sure they don't get auto-merged.